lib/index.js

```diff
@@ -104,23 +104,18 @@ internals.implementation = function(server, options) {
       scheme: scheme,
       attributes: attributes,
     };
-    let _errorType = errorType; // copies of params
-    let _message = message; // so we can over-write them below
-    let _scheme = scheme; // without a linter warning
-    let _attributes = attributes; // if you know a better way please PR!
 
-    if (options.errorFunc && internals.isFunction(options.errorFunc)) {
+    if (internals.isFunction(options.errorFunc)) {
```

Comment:
The idea behind having this check is that we test for presence of the key before testing for type.
But you're _right_, the `internals.isFunction` does this check already, so OK to remove. 👍